Drukhari/Readme.md

```diff
@@ -0,0 +1,96 @@
+  * **`Overlord (Empowered):`** *Leader Ability* <br> Enable the profile ability `Re-roll all possible failed wound rolls (if attacker)`.
+
+* #### Beastmaster:
+*This unit has different values for its defensive characteristics, and hence is split into separate profiles. Alternatively, there is a profile where the entire unit is housed in a single profile, though the defensive characteristics can not be simulated.*
+  * `Beastmaster (unit)`
+  * `Beastmaster x1`
+  * `Clawed Fiends x1`
+  * `Khymerae x2`
+  * `Razorwing Flocks x3`
+  * `** Beastmaster (Ability):`** Enable the profile abliity `Re-roll all hit roll results of 1 (if attacker)`.
+
+* #### Court of the Archon:
+  * **`Lhamaean:`** <br> Enable the profile ability `LETHAL HITS`.
+  * **`Sslyth:`** <br> Enable the profile ability `-1 to wound roll (if defender)`.
+
+* #### Court of the Archon + Archon:
+  * **`Overlord:`** *Leader Ability* <br> Enable the profile ability `Re-roll all wound roll results of 1 (if attacker)`.
+  * **`Overlord (Empowered):`** *Leader Ability* <br> Enable the profile ability `Re-roll all possible failed wound rolls (if attacker)`.
+  * **`Lhamaean:`** <br> Enable the profile ability `LETHAL HITS`.
+  * **`Sslyth:`** <br> Enable the profile ability `-1 to wound roll (if defender)`.
+
+* #### Drazhar:
+  * **`Master of Blades:`** *Leader Ability* <br> Enable the profile ability `+1 to wound roll (if attacker)`.
+
+* #### Haemonculus:
+  * **`Master of Pain:`** *Leader Ability* <br> Enable the profile ability `Feel no pain 4+`.
+
+* #### Kabalite Warriors x10 + Archon:
+  * **`Overlord:`** *Leader Ability* <br> Enable the profile ability `Re-roll all wound roll results of 1 (if attacker)`.
+  * **`Overlord (Empowered):`** *Leader Ability* <br> Enable the profile ability `Re-roll all possible failed wound rolls (if attacker)`.
+
+* #### Incubi x5 + Drazhar:
+  * **`Master of Blades:`** *Leader Ability* <br> Enable the profile ability `+1 to wound roll (if attacker)`.
+
+* #### Lelith Hesperax:
+  * **`Brides of Death:`** *Leader Ability* <br> Enable the profile ability `+1 to strength (melee only) (if attacker)`.
+  * **`Brides of Death:`** *Leader Ability* <br> Enable the profile ability `Improve AP by 1 (melee only) (if attacker)`.
+  * **`Thrilling Spectacle:`** <br> Enable the profile ability `3++ invuln save`.
+  * **`Thrilling Spectacle:`** <br> Use the weapon profile `Lelith's blades (Thrilling Spectacle)`.
+
+* #### Ravager:
+  * **`Eradicate the Foe:`** <br> Enable the profile ability `Re-roll all hit roll results of 1 (if attacker)`.
+
+* #### Razorwing Jetfighter:
+  * **`Ground Attack Craft:`** <br> Enable the profile ability `+1 to hit roll (if attacker) (excludes FLY)`.
+
+* #### Reavers:
+  * **`Grav-talon:`** <br> Enable the profile ability `+1 to wound roll (if attacker charged) (melee only) (if attacker)`.
+
+* #### Succubus:
```

Comment:
* `Critical hit : 1 extra hit (if attacker)` ability needs the `melee only` condition
* Missing `Character` Keyword.